preprocessors/spatial-index.preprocessor.js

```diff
@@ -34,6 +34,7 @@ module.exports = function(infile, outdir, callback) {
         .on('exit', function() {
           // If error printed to --validate-features log
           if (data.indexOf('Error') != -1) {
+            console.log(data);
```

Comment:
@GretaCB what's the plan of attack?
This has changed here (https://github.com/mapbox/preprocessorcerer/compare/stderr-fix#diff-39694ce4a586caad3e6ad813c295b454R32) too as latest `mapnik-index` is now returning an error code, if `--validate-features` fails.
Need to wait to update to latest node-mapnik?